3.12/graphs-general-graphs.md

```diff
@@ -7,16 +7,60 @@ General Graphs
 ==============
 
 This chapter describes the [general-graph](graphs.html) module.
-It allows you to define a graph that is spread across several edge and document collections.
-This allows you to structure your models in line with your domain and group them logically in collections giving you the power to query them in the same graph queries.
-There is no need to include the referenced collections within the query, this module will handle it for you.
+It allows you to define a graph that is spread across several edge and document
+collections.
+This allows you to structure your models in line with your domain and group
+them logically in collections giving you the power to query them in the same
+graph queries.
+There is no need to include the referenced collections within the query, this
+module will handle it for you.
 
 New to ArangoDB? Take the free
 [ArangoDB Graph Course](https://www.arangodb.com/arangodb-graph-course){:target="_blank"}
 for freshers.
 
-Three Steps to create a graph
------------------------------
+## Getting started
+
+### Create a GeneralGraph using the Web Interface
+
+The Web Interface (also called Web UI) allows you to easily create and manage
+GeneralGraphs. To get started, follow the steps outlined below.
+
+1. In the main page of the Web Interface, go to the left sidebar 
+   menu and select the **Graphs** tab.
+2. To add a new graph, click **Add Graph**.
```

Comment:
Here I call it "card": https://main--zealous-morse-14392b.netlify.app/docs/3.11/graphs-traversals-example-data.html
Should we consistently do that or not mention the user interface element at all?

Reply:
Didn't mention the element in this particular case, but generally speaking let's go with "card".